apps/admin_panel/assets/src/websocket/connector.js

```diff
@@ -0,0 +1,3 @@
+import Socket from 'phoenix-socket'
+const socket = new Socket('wss://localhost:4000/api/socket', { params: { userToken: '123' } })
```

Comment:
Should probably be `/api/admin/socket` btw. Not sure why it's in this PR tho :D

Reply:
mistake lol it should be in another PR